tests/tests/src/core/Core_NullableTests.mjs

```diff
@@ -7,7 +7,7 @@ function shouldHandleNullableValues() {
   let tUndefined = undefined;
   let tValue = "hello";
   let tmp;
-  tmp = tNull === null ? true : false;
+  tmp = tNull === null || tNull === undefined ? tNull === null : false;
```

Comment:
The tNull === undefined was already gone in #7138?

Reply:
I'll merge and rebase directly on https://github.com/rescript-lang/rescript-compiler/pull/7138